<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodaj Wpis</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QPushButton{
	background-color: rgb(255,248,222);
	border: 2px solid rgb(153,169,143);
	border-radius: 10px;
	padding: 3px
}	
.QPushButton:hover {
	background-color: rgb(201,219,178);
	border: 2px solid rgb(153,169,143);
}
*{
	background-color: rgb(227,242,193);
	color: rgb(0, 0, 0);
	font:  &quot;Century Gothic&quot;;
}
button{
	font:  &quot;Century Gothic&quot;;}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="centralWidget" native="true">
     <property name="styleSheet">
      <string notr="true">*{
	
	background-color: rgb(246,255,222);
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextEdit" name="entryTxt"/>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="entryDT"/>
      </item>
      <item>
       <widget class="QFrame" name="optionsFrm">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="sadChk">
           <property name="text">
            <string>Smutne</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="neutralChk">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Neutralne</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cheerfulChc">
           <property name="text">
            <string>Radosne</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
